scripts/deploy/npm-start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+cd /home/gcc_website
+npm start
```

Comment:
Whoops, I juts realized this, but "npm start" is for development mode.

We want to add the following to `npm-install.sh`:
- npm run babel
- npm run webpack
- npm run test
- npm run esdoc

And then `npm-start.sh` should instead contain:
 - npm run serve

We'll need to ssh into the EC2 instance, create a `config.json`, and populate it with the correct config variables as well. Check `src/config.js` to see what's needed